src/transaction/log_page_buffer.c

```diff
@@ -6240,6 +6274,21 @@ logpb_remove_archive_logs (THREAD_ENTRY * thread_p, const char *info_reason)
 	  min_arv_required_for_cdc--;
 	  last_deleted_arv_num = MIN (last_deleted_arv_num, min_arv_required_for_cdc);
 	}
+
+      /* flashback */
+      if (flashback_is_needed_to_keep_archive ())
+	{
+
+	  flashback_first_pageid = flashback_min_log_pageid_to_keep ();
+
+	  /* NULL check for flashback_first_pageid is done in flashback_is_needed_to_keep_archive () */
+	  if (logpb_is_page_in_archive (flashback_first_pageid))
+	    {
+	      min_arv_required_for_flashback = logpb_get_archive_number (thread_p, flashback_first_pageid);
+	      min_arv_required_for_flashback--;
+	      last_deleted_arv_num = MIN (last_deleted_arv_num, min_arv_required_for_flashback);
```

Comment:
Please add a debugging log here too. And It would be nice to add a prefix that can be distinguished from the log left by the daemon.

Reply:
OK, I will add debug message here too.